### PR TITLE
Tests file created

### PR DESCRIPTION
**Pull request by Abraham12611**
I created a new `tests` folder and moved the unittests file.

### CODE REVIEW

**Review by tom-halpin — CHANGES_REQUESTED** [review]
Following the move the tests are no longer working, also please include a CHANGELOG.md entry describing the change

To get the tests working in the new folder something like the following updates to the test script code are needed to fix the import statement for the class being tested

```python

import os
import sys
import unittest
from unittest.mock import patch, MagicMock
import openai

# Add the root folder to the Python module search path
sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))

from openAiCompletion import openAiCompletion
```

**Comment by Abraham12611** [comment]
Made requested changes

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.2.0] - 2023-06-06
```
Comment (tom-halpin) [inline CHANGELOG.md:8]:
```suggestion
## [1.2.1] - 2023-06-06
```

Comment (tom-halpin) [reply CHANGELOG.md:8]: Please update the version number in the change log

**Review by tom-halpin — CHANGES_REQUESTED** [review]
Thanks one minor change in the CHANGELOG.md required see review for details, there is also a pylint error reported in the corresponding GitHub action which you should address

**Comment by tom-halpin** [comment]
Thank you @Abraham12611

**Pull request merged**